Source/WebCore/inspector/agents/page/PageNetworkAgent.cpp

```diff
@@ -45,7 +45,7 @@ using namespace Inspector;
 WTF_MAKE_TZONE_ALLOCATED_IMPL(PageNetworkAgent);
 
 PageNetworkAgent::PageNetworkAgent(PageAgentContext& context, InspectorClient* client)
-    : InspectorNetworkAgent(context)
+    : InspectorNetworkAgent(context, client->maximumResourcesContentSize())
```

Comment:
rather than grabbing from the `InspectorClient` now that we have a preference i think we could just do this
```cpp
: InspectorNetworkAgent(context, context.inspectedPage.settings().inspectorMaximumResourcesContentSize())
```